src/components/views/emojipicker/EmojiPicker.tsx

```diff
@@ -27,6 +27,7 @@ import Preview from "./Preview";
 import QuickReactions from "./QuickReactions";
```

Comment:
This encourages creating more detailed, personalized reactions, so maybe there should be a reminder in the menu that reactions are never encrypted.